### PR TITLE
feat(docs/op-stack): Freeze monorepo links to the current commit

### PR DESCRIPTION
**Pull request by qbzzt**
Change the links to the monorepo from links to the `develop` branch to links to the current commit. This way, when something changes in the code, we aren't pointing to the wrong line or the wrong file.

Closing: DEVRL-932

### CODE REVIEW

File: docs/op-stack/src/docs/releases/bedrock/explainer.md [hunk 1]
```diff
@@ -216,7 +216,7 @@ There are two fundamental reasons for having any diff at all: handling deposited
 
 #### Handling deposited transactions
 
-To represent [deposited transactions](#arbitrary-message-passing-from-l1) in the rollup, there is an additional transaction type introduced. The [execution client](#execution-client) implements this [new transaction type](https://github.com/ethereum-optimism/optimism/blob/develop/specs/%5Bdeposits%5D(#deposits).md%23the-deposited-transaction-type) according to the [EIP-2718 typed transactions](https://eips.ethereum.org/EIPS/eip-2718) standard.
+To represent [deposited transactions](#arbitrary-message-passing-from-l1) in the rollup, there is an additional transaction type introduced. The [execution client](#execution-client) implements this [new transaction type](https://github.com/ethereum-optimism/optimism/blob/d69cb12f6dcbe3d5355beca8997fbac611b7fe37/specs/%5Bdeposits%5D(#deposits).md%23the-deposited-transaction-type) according to the [EIP-2718 typed transactions](https://eips.ethereum.org/EIPS/eip-2718) standard.
```
Comment (sbvegan) [inline docs/op-stack/src/docs/releases/bedrock/explainer.md:219]: this link is broken

Comment (qbzzt) [reply docs/op-stack/src/docs/releases/bedrock/explainer.md]: Thank you, resolved.

**Comment by sbvegan** [comment]
Looks good to me!

There's been an update to GitHub, so I can't hit the "approve" button anymore (until I'm granted access).

**Review by sbvegan — APPROVED** [review]

**Comment by qbzzt** [comment]
I tried to put this PR on top of the existing code to fix the semgrep issues. It failed, so I think the fastest solution would be to create a new branch.

**Pull request merged**